src/hacbs/components/PipelineRunDetailsView/tabs/PipelineRunDetailsTab.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import {
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListDescription,
+  Title,
+} from '@patternfly/react-core';
+import { calculateDuration } from '../../../utils/pipeline-utils';
+
+type PipelineRunDetailsTabProps = {
+  pipelineRun: any;
+};
+
+const PipelineRunDetailsTab: React.FC<PipelineRunDetailsTabProps> = ({ pipelineRun }) => {
+  const duration = calculateDuration(
+    typeof pipelineRun.status?.startTime === 'string' ? pipelineRun.status?.startTime : '',
+    typeof pipelineRun.status?.completionTime === 'string'
+      ? pipelineRun.status?.completionTime
+      : '',
+  );
+  return (
+    <>
+      <Title headingLevel="h4" className="pf-c-title pf-u-mt-lg pf-u-mb-lg">
+        {' '}
+        Pipelinerun details
+      </Title>
```

Comment:
```suggestion
      <Title headingLevel="h4" className="pf-c-title pf-u-mt-lg pf-u-mb-lg">
        Pipelinerun details
      </Title>
```

Reply:
Done, not sure why VS code adds always adds these by default on 'format document'

Reply:
If you add a space between the closing arrow and text `> Foo`, prettier will format the text to the next line and add `{' '}` as it tries to preserve your spacing.